MMOCoreORB/bin/scripts/mobile/spawn/tatooine_world.lua

```diff
@@ -1315,6 +1315,15 @@ tatooine_world = {
 			weighting = 2,
 			size = 35
 		},
+		{
+			lairTemplateName = "tatooine_krayt_dragon_ancient_neutral_none",
+			spawnLimit = 5,
+			minDifficulty = 325,
+			maxDifficulty = 340, 
+			numberToSpawn = 0,
+			weighting = 4,
```

Comment:
Should ancients be more common than other krayt types? I'd think this should at least be 2 to make them equally as rare as a grand spawn, if not 1 to make them rarer.